app/views/tag/show/_header.html.erb

```diff
@@ -140,4 +140,10 @@
 .bottom-right a {
   color: #fff;
 }
+
+@media (min-width: 576px) {
+.d-md-block {
```

Comment:
Properly indent the `.d-md-block` rule.

Reply:
@VladimirMikulic I dont know the .d-md-block rules can you please guide me.

Reply:
Alright, this is your CSS:
```css
@media (min-width: 576px) {
.d-md-block {
    display: block !important;
}
}
```

This is how it should be formatted:

```css
@media (min-width: 576px) {
    .d-md-block {
        display: block !important;
    }
}
```